dsgrid/project.py

```diff
@@ -161,7 +162,8 @@ def process_query(self, context: QueryContext):
             for df in dfs[1:]:
                 if df.columns != main_df.columns:
                     raise Exception(
-                        f"BUG: dataset columns must match. main={main_df.columns} new={df.columns}"
+                        f"BUG: dataset columns must match. main={main_df.columns} new={df.columns} "
+                        f"diff={set(main_df.columns).symmetric_difference(df.columns)}"
```

Comment:
Add `set()` to df.columns?

Reply:
Interesting, I just realize `set().some_set_funct(list)` is the format used everywhere else too, and it works.